db/schema.rb

```diff
@@ -85,6 +85,8 @@
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
     t.string   "slug"
+    t.boolean  "status"
```

Comment:
Status is whether to decide if it is active or inactive.  Approval is if a platform admin has approved it or not. Each is a unique state, and i feel two columns gives us maximum flexibility.